thoth_dependency_monkey/validation_dao.py

```diff
@@ -140,6 +183,29 @@ def create(self, data):
         if v['ecosystem'] not in ECOSYSTEM:
             raise EcosystemNotSupportedError(v['ecosystem'])
 
+        packages = v['stack_specification']
+        packages = packages.split("//")
+        package_versions = dict()
+        for name in packages:
+            self._pckg_name = re.findall("[a-zA-Z]+", name)
+            #checks if package name not entered
+            if not self._pckg_name:
+                break
+            #if incorrect version number or character,default is >0
+            self._ver_no = re.findall(r'\s*([\d.]+)', name)
+            if not ver_no:
+                ver_no = "0"
+            else:
+                ver_no = ver_no[0]
+            self._char = re.sub("[\w+.]", "", name)
+            if not self._char:
+                char = ">"
+            temp_dict = dict()
+            temp_dict = {pckg_name[0]:get_all_versions(self._pckg_name[0],self._ver_no,self._char)}
+            package_versions = {**package_versions,**temp_dict}
+            my_list = [dict(zip(package_versions,v)) for v in product(*package_versions.values())]
```

Comment:
> BTW could you adjust algorithm in a way that it would submit not all stacks but starting Nth to Kth?

Hint: [islice](https://docs.python.org/3.6/library/itertools.html#itertools.islice)